Modules/Loadable/SubjectHierarchy/qSlicerSubjectHierarchyModule.cxx

```diff
@@ -116,10 +121,20 @@ void qSlicerSubjectHierarchyModule::setup()
 {
   this->Superclass::setup();
 
-  if (qSlicerApplication::application())
+  qSlicerApplication * app = qSlicerApplication::application();
+  if (app)
     {
+    // Register settings panel
     qSlicerSubjectHierarchySettingsPanel* panel = new qSlicerSubjectHierarchySettingsPanel();
-    qSlicerApplication::application()->settingsDialog()->addPanel("Subject hierarchy", panel);
+    app->settingsDialog()->addPanel("Subject hierarchy", panel);
+
+    // Register parameterNodeWrapper plugins
+#ifdef Slicer_USE_PYTHONQT
+    if (!qSlicerCoreApplication::testAttribute(qSlicerCoreApplication::AA_DisablePython))
+      {
+      app->pythonManager()->executeString(QString("import SubjectHierarchyLib.parameterNodeWrapper"));
```

Comment:
Would it be possible to just import SubjectHierarchyLib and SubjectHierarchyLib would know what to do exactly as part of that? It would simplify the code if the C++ class would not need to know any internal details of SubjectHierarchyLib.

Reply:
I was initially hesitant and went forward with being explicit. That said, your comment

> simplify the code if the C++ class would not need to know any internal details

convinced me the simplification was worth it.